src/summary_writer.jl

```diff
@@ -28,28 +28,45 @@ Arguments:
 * logdir: A string. Directory where event file will be written.
 * graph: A `Graph` object.
 """
-function FileWriter(log_dir::AbstractString; graph=get_def_graph())
+function FileWriter(log_dir::AbstractString; graph=nothing)
+    if !tf.in_eager_mode() && graph===nothing
+        graph = get_def_graph()
+    end
     mkpath(log_dir)
-    path = joinpath(log_dir, "events")
-    pyo = @py_proc pywrap_tensorflow[][:EventsWriter](py_bytes($path))
-    writer = FileWriter(pyo, String(log_dir))
+    local path
+    for i in Iterators.countfrom(1)
+        path = joinpath(log_dir, "events.out.tfevents.$i")
+        isfile(path) || break
+    end
+    rm(path, force=true)
```

Comment:
```suggestion
```
Do not want this,
we've already proven there is no file of that name,
and if its a dir this won't remove it anyway since not recursive.